src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -8284,11 +8316,22 @@ private LabeledStatementSyntax ParseLabeledStatement()
         }
 
         /// <summary>
-        /// Parses any kind of local declaration statement: local variable, local funcion, or let statement.
+        /// Parses any kind of local declaration statement: local variable, local function, let statement, or deconstruction declaration.
```

Comment:
I think we don't support "let statements" yet. Consider removing reference to them in the comment if you will be making other changes to the file in this PR.